<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Age:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender(M/W):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Works(No/Yes)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Salary:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vacation days:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSalary">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditVacationDays">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_struct1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Struct 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_struct3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Struct 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_struct4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Struct 4:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_struct5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Struct 5:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_struct2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Struct 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="s1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>661</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="s2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>661</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="s3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>661</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="s4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>661</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="s5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>661</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>InitialiseStructure</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Count of projects:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxGender">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Man</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Woman</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxIsWork">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
